ui/app/routes/allocations/allocation.js

```diff
@@ -36,18 +36,18 @@ export default class AllocationRoute extends Route.extend(WithWatchers) {
     }
   }
 
-  model() {
-    // Preload the job for the allocation since it's required for the breadcrumb trail
-    return super
-      .model(...arguments)
-      .then((allocation) => {
-        const jobId = allocation.belongsTo('job').id();
-        return this.store
-          .findRecord('job', jobId)
-          .then(() => this.store.findAll('namespace')) // namespaces belong to a job and are an asynchronous relationship so we can peak them later on
-          .then(() => allocation);
-      })
-      .catch(notifyError(this));
+  async model() {
+    try {
+      // Preload the job for the allocation since it's required for the breadcrumb trail
+      const allocation = await super.model(...arguments);
+      const jobId = allocation?.belongsTo('job').id();
```

Comment:
@philrenaud based on the test suite, I believe that we actually need to throw an error if we don't find the allocation so using the [elvis operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining), will fail a test and not propagate the error message we're expecting. Will revert this.

Reply:
Meh.... there's more to the story than just this comment though... slight change that using the `try/catch` block and `await` syntax is causing something funky.